project/Doc.scala

```diff
@@ -31,7 +31,30 @@ object Scaladoc extends AutoPlugin {
       scalacOptions in Compile <++= (version, baseDirectory in ThisBuild) map scaladocOptions,
       autoAPIMappings := CliOptions.scaladocAutoAPI.get
     )) ++
-    Seq(validateDiagrams in Compile := true) ++
+    Seq(
+      validateDiagrams in Compile := true,
+      apiMappings ++= {
+        // Based on https://github.com/ThoughtWorksInc/sbt-api-mappings
+        val IvyRegex = """^.*[/\\]([\.\-_\w]+)[/\\]([\.\-_\w]+)[/\\](?:jars|bundles)[/\\]([\.\-_\w]+)\.jar$""".r
+
+        for {
+          jar <- (dependencyClasspath in Compile in doc).value.toSet ++ (dependencyClasspath in Test in doc).value
+          fullyFile = jar.data
+          url <- fullyFile.getCanonicalPath match {
+            case IvyRegex(organization, name, jarBaseFile) if jarBaseFile.startsWith(s"$name-") => {
+              val version = jarBaseFile.substring(name.length + 1, jarBaseFile.length)
+              organization match {
+                case "com.typesafe.akka" =>
```

Comment:
And `org.reactivestreams` => http://www.reactive-streams.org/reactive-streams-1.0.0-javadoc/